package.json

```diff
@@ -33,6 +33,7 @@
     "api-stubs:reset": "npx ts-node --transpile-only ./wiremock/resetStubs.ts",
     "start-test-wiremock": "docker-compose -f docker-compose-test.yml up -d",
     "generate-types": "script/generate-types",
+    "form-page:generate": "npx ts-node --transpile-only ./server/form-pages/utils/generator.ts",
```

Comment:
The generator has somewhat gone out of whack with the way form pages work now, so you may want to get rid of this commit